modules/social_features/social_follow_content/src/Controller/Overview.php

```diff
@@ -0,0 +1,206 @@
+<?php
+
+namespace Drupal\social_follow_content\Controller;
+
+use Drupal\Component\Utility\Unicode;
+use Drupal\Core\Controller\ControllerBase;
+use Drupal\Core\Url;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\HttpFoundation\RequestStack;
+
+/**
+ * Class Overview.
+ *
+ * @package Drupal\social_follow_content
+ */
+class Overview extends ControllerBase {
```

Comment:
What is the reason that we're creating a new controller here? Why does the view not fit our purposes?

The view already loads the flags, so the view base table is not the problem. We could just add a post relationship and load the posts when a flag references one. Then the fields we display would show both the post and the content title (as only one is ever available).

The controller is very inflexible. If a site has decided to display 30 or 10 lines on a page then that is now no longer changeable by a site manager. The controller does no more then what views was designed to do.

Any changes that a site builder has made to the content overview (add an author field for example) will also be thrown out and require coding to get back.

Reply:
@Kingdutch I used the controller because Views table sorting does not work correctly when using few entity types.